src/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -542,6 +542,18 @@ public virtual X509Certificate RemoteCertificate
             }
         }
 
+        [CLSCompliant(false)]
+        public virtual TlsCipherSuite? NegotiatedCipherSuite
+        {
+            get
+            {
+                CheckThrow(true);
+                SslConnectionInfo info = _context.ConnectionInfo;
+
+                return info == null ? (TlsCipherSuite?)null : info.TlsCipherSuite;
```

Comment:
Couldn't this just be:
```C#
return info?.TlsCipherSuite;
```
?

Reply:
since we removed nullable cannot do it anymore